_posts/blog/2017-02-22-team_cut_story.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: post
+title: "A Team Cut Story"
+description:
+modified:
+categories: blog
+author: johannes_theilmann
+excerpt: A short story about how to measure a team cut experiment. We change teams a few times already – this article shows how can we learn which set up was actually more productive and how to perform those experiments as smoothly as possible.
+tags: [Team, Story, Agile]
+image:
+date: 2017-02-22T00:00:00-04:00
+---
+At upday we challenge ourselves to achieve technical excellence and a strong understanding of business needs. Due to external and internal circumstances we have quite a volatile team setup. At least each quarter we change either the team members, the PO or the whole structure of our teams - from horizontal to cross-functional to fluent and back again. However our goal is to have stable teams with a clear business orientation while also enabling team members to grow technically and to support each other (eg. reviews, testing, tech discussions).
+
+It is the challenge for each team member to adapt quickly to new teams, skills and goals. I would like to share how we approach building stronger teams and to have a checklist for myself when a new team is set up:
+
+**As a** Product Owner **I want** to set up teams quickly and efficiently in order to get them up ‘n’ running fast and deliver the highest value.
+
+Acceptance Criteria:
```

Comment:
Using Present in acceptance criteria doesn't feel right (next bullet points). Times are mixed anyway.